crates/walrus-service/src/event/event_processor/catchup.rs

```diff
@@ -35,6 +35,8 @@ struct DownloadedBlob {
     epoch: Epoch,
     first_event_index: Option<IndexedStreamEvent>,
     events: Vec<IndexedStreamEvent>,
+    start_checkpoint: u64,
```

Comment:
Question: do we want to include `start_checkpoint` if it is not used?

Reply:
Yeah, i think this is for logging purposes, it's probably better to have it.